nin/frontend/app/styles/common.less

```diff
@@ -530,8 +530,29 @@ input[type=range]:focus::-ms-fill-upper {
   padding-bottom: 10px;
   font-family: monospace;
   z-index: 999999999999999;
-  background-color: rgba(113, 75, 75, 0.6);
-  overflow-y: scroll;
```

Comment:
Is there a way to get the error to still show the line number? In your example it has wrapped outside the screen. Would it be better if the muted part of he error could wrap over multiple lines, slightly indented?